po/es.po

```diff
@@ -0,0 +1,706 @@
+
+#: src/info.rs:71 src/info.rs:118
+msgid "Last Modified"
+msgstr "Modificado por última vez"
+
+#: src/info.rs:72 src/info.rs:120
+msgid "Out Of Date"
+msgstr "Desactualizado"
+
+#: src/info.rs:73 src/info.rs:125
+msgid "ID"
+msgstr "ID"
+
+#: src/info.rs:74 src/info.rs:126
+msgid "Package Base ID"
+msgstr "ID Base del Paquete"
+
+#: src/info.rs:75 src/info.rs:127
+msgid "Keywords"
+msgstr "Palabras clave"
+
+#: src/info.rs:76 src/info.rs:129
+msgid "Snapshot URL"
+msgstr "URL de Instantánea"
+
+#: src/info.rs:92
+msgid "No"
+msgstr "No"
+
+#: src/info.rs:112
+msgid "Optional Deps"
+msgstr "Dependencias Opcionales"
+
+#: src/install.rs:131 src/install.rs:545 src/install.rs:944
+msgid "no architecture"
+msgstr "sin arquitectura"
+
+#: src/install.rs:138
+msgid "can't build package as root"
+msgstr "no se puede construir paquete como root"
+
+#: src/install.rs:142
+msgid "failed to parse srcinfo generated by makepkg"
+msgstr "error al parsear srcinfo generado por makepkg"
+
+#: src/install.rs:171 src/install.rs:442
+msgid "Resolving dependencies..."
+msgstr "Resolviendo dependencias..."
+
+#: src/install.rs:220 src/install.rs:225
+msgid "failed to download sources"
+msgstr "error al descargar fuentes"
+
+#: src/install.rs:230 src/install.rs:246 src/install.rs:254
+msgid "failed to build"
+msgstr "error al construir"
+
+#: src/install.rs:233
+msgid "parsing pkg list..."
+msgstr "parseando lista pkg..."
+
+#: src/install.rs:261 src/install.rs:1462
+msgid "{}-{} is up to date -- skipping build"
+msgstr "{}-{} esta actualizado -- saltando construcción"
+
+#: src/install.rs:333 src/query.rs:50 src/lib.rs:84
+msgid "error:"
+msgstr "error:"
+
+#: src/install.rs:334
+msgid "could not get news"
+msgstr "no se pudieron obtener noticias"
+
+#: src/install.rs:339 src/install.rs:521 src/install.rs:716 src/install.rs:848
+msgid "Proceed with installation?"
+msgstr "¿Proceder con la instalación?"
+
+#: src/install.rs:356
+msgid "no targets specified (use -h for help)"
+msgstr "no se han seleccionado objetivos (usa -h para ayuda)"
+
+#: src/install.rs:434 src/install.rs:461 src/search.rs:324 src/search.rs:360
+#: src/lib.rs:227
+msgid " there is nothing to do"
+msgstr " nada por hacer"
+
+#: src/install.rs:488
+msgid "can't install AUR package as root"
+msgstr "no se puede instalar un paquete de AUR como root"
+
+#: src/install.rs:507
+msgid "Remove make dependencies after install?"
+msgstr "¿Eliminar dependencias de make tras la instalación?"
+
+#: src/install.rs:518
+msgid "Proceed to review?"
+msgstr "¿Proceder a revisión?"
+
+#: src/install.rs:559
+msgid "The following packages are not compatible with your architecture:"
+msgstr "Los siguientes paquetes no son compatibles con tu arquitectura:"
+
+#: src/install.rs:573
+msgid "Would you like to try build them anyway?"
+msgstr "¿Quieres volver a intentar construirlos igualmente?"
+
+#: src/install.rs:662
+msgid "packages failed to build: {}"
+msgstr "paquetes que no se pudieron construir: {}"
```

Comment:
```suggestion
msgstr "no se pudo construir los siguentes paquetes: {}"
```

Reply:
I'm pretty sure it should be "pudieron" since you are refereing to multiple things

Reply:
Ah, I suppose since it's reflexive that's true.